cli/wallet/ledger/ledger.go

```diff
@@ -38,6 +38,10 @@ func (wf *ledgerWalletFactory) Kind() string {
 	return Kind
 }
 
+func (wf *ledgerWalletFactory) PrettyKind(config map[string]interface{}) string {
+	return wf.Kind()
```

Comment:
Could include `cfg.Derivation` and `cfg.Number` though?

Reply:
Good idea! Added.